distributions/rhel-85.json

```diff
@@ -0,0 +1,15 @@
+{
+  "module_platform_id": "platform:el8",
+  "distribution": {
+    "name": "rhel-85",
+    "description": "Red Hat Enterprise Linux (RHEL) 8"
+  },
+  "x86_64": {
+    "image_types": [ "ami", "vhd", "edge-commit" ],
```

Comment:
I think this should just be added as part of 8.4 instead, no?

Reply:
if we just want edge support in GA then we can just add it to the current rhel-8 file

Reply:
GA?